### PR TITLE
Rename workflow to process everywhere

### PR DESCRIPTION
**Pull request by npepinpe**
**Description**

This PR adapts menski's script provided in https://github.com/camunda-cloud/zeebe/pull/6455 to rename every instance of workflow to process.

The only thing I had to change was the dependency on `zeebe-workflow-generator` which I suppose will stay as is for now.

The other failures are because we use the client, which still refers to workflows, but there has been no release with process yet - so maybe we do an alpha release a little earlier to get that.

### CODE REVIEW

**Comment by npepinpe** [comment]
Last issue is that we depend on zeeqs for integration testing, but zeeqs still uses the old API I guess :slightly_smiling_face:

**Comment by Zelldon** [comment]
Plus zeebe-chaos will not work (it still uses workflowInstance as well)

**Comment by korthout** [comment]
rebased on develop branch

**Comment by korthout** [comment]
@pihme Care to take a look at this? I've rebased and added some extra stuff.

**Comment by korthout** [comment]
@npepinpe Please also have a look :)

File: pom.xml [hunk 1]
```diff
@@ -428,7 +428,7 @@
     <version.slf4j>1.7.30</version.slf4j>
     <version.vavr>0.10.3</version.vavr>
     <version.workflowgenerator>1.0.2</version.workflowgenerator>
-    <zeebe.version>0.26.0</zeebe.version>
+    <zeebe.version>1.0.0-alpha4</zeebe.version>
```
Comment (npepinpe) [inline pom.xml:431]:
```suggestion
    <zeebe.version>1.0.0-alpha5</zeebe.version>
```

File: processes/test-notification.bpmn [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (npepinpe) [inline processes/test-notification.bpmn:1]: I guess we can remove this? I think I used it at some point to test notifications. I must've added it by mistake, sorry

**Review by npepinpe — COMMENTED** [review]
Looks good to me in general. I take it we disabled the IT tests...maybe depending on zeeqs is the best, as Zeebe will increasingly depend on Testbench in the future, but there are no similar plans for zeeqs (there are some vague plans to have a query API one day though, so that would replace it - I imagine we can wait until more comes out of this)

**Comment by npepinpe** [comment]
As I'm the author, I'm not allowed to approve this PR :smile:

**Review by pihme — APPROVED** [review]

File: processes/test-notification.bpmn (hunk 2 above)
Comment (korthout) [reply processes/test-notification.bpmn]: Removed it from your commit

**Comment by korthout** [comment]
bors merge

**Pull request merged**